tests/datasets/test_ShapeNet.py

```diff
@@ -63,17 +64,22 @@ def test_Voxels():
 
     shutil.rmtree('tests/datasets/cache/voxels')
 
-
-# def test_Images():
-#     images = shapenet.ShapeNet_Images(root=SHAPENET_ROOT, cache_dir=CACHE_DIR,
-#                                       categories=['phone'], views=1, train=True, split=.7)
-#     assert len(images) == 736
-#     for obj in images:
-#         assert set(obj['data']['images'].shape) == set([137, 137, 4])
-#         assert os.path.isfile(obj['attributes']['name'] + '/rendering/00.png')
-#         assert set(obj['data']['params']['cam_mat'].shape) == set([3, 3])
-#         assert set(obj['data']['params']['cam_pos'].shape) == set([3])
-
+@pytest.mark.parametrize('categories', [['chair'], ['plane', 'bench', 'cabinet', 'car', 'chair',
+                                                    'monitor', 'lamp', 'speaker', 'rifle',
+                                                    'sofa', 'table', 'phone', 'watercraft']])
+@pytest.mark.skipif(not Path(SHAPENET_RENDERING_ROOT).exists(), reason=REASON)
```

Comment:
`REASON` is specific to ShapeNet and SHAPENET_ROOT. Should be changed here to be specific to shapenet rendering.

Reply:
Good catch, fixed